src/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -62,17 +64,19 @@ void fill_data_with_broadcast(InferenceEngine::Blob::Ptr &blob, InferenceEngine:
  * @param values data to broadcast
  */
 void fill_data_with_broadcast(InferenceEngine::Blob::Ptr &blob, size_t axis, std::vector<float> values);
-
+void fill_data_with_broadcast(ov::Tensor& tensor, size_t axis, std::vector<float> values);
 /**
  * Make a view blob with new shape. It will reinterpret original tensor data as a tensor with new shape.
  *
  * NB! Limitation: the nwe one blob will no have ownership of data buffer. The original blob should be alive
  *     while view is in use.
  *
- * @param blob original source tensor
+ * @param tensor original source tensor
  * @param new_shape new one shape for view blob
  * @return new one blob view
  */
+ov::Tensor make_reshape_view(const ov::Tensor &tensor, std::vector<size_t> new_shape);
```

Comment:
Do we really need it in `data_utils`? Look like local method used for 1 or 2 tests

Reply:
This method is used in cpu func and gpu func tests. So I prefer to keep it here.